webapp/views/App/views/Users/UserInvite/store/hooks.js

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useReducer } from 'react'
+import { useHistory } from 'react-router'
+import { useDispatch } from 'react-redux'
+import * as R from 'ramda'
+import axios from 'axios'
+
+import * as UserInvite from '@core/user/userInvite'
+
+import { useSurveyCycleKey, useSurveyId } from '@webapp/store/survey'
+import { LoaderActions, NotificationActions } from '@webapp/store/ui'
+import * as Validation from '@core/validation/validation'
+
+import { appModuleUri, userModules } from '@webapp/app/appModules'
+
+import * as actions from './actions'
+import reducer from './reducer'
+import initialState from './initialState'
+
+export const useInviteUser = () => {
+  const [state, dispatch] = useReducer(reducer, initialState)
+  const dispatchRedux = useDispatch()
+  const history = useHistory()
+
+  const surveyId = useSurveyId()
+  const surveyCycleKey = useSurveyCycleKey()
+
+  useEffect(() => {
+    actions.resetUserInvite()(dispatch)
+  }, [])
```

Comment:
since it is now refactored using local state, reset state onMount is useless I suppose